ferrilata-devilline/Services/Extensions/AdminJsonAnalizer.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ferrilata_devilline.Services.Extensions
+{
+    public static class AdminJsonAnalizer
+    {
+        private class ExpectedFormat
+        {
+            [JsonProperty(Required = Required.Always)]
+            string version { set; get; }
+
+            [JsonProperty(Required = Required.Always)]
+            string name { set; get; }
```

Comment:
As per MS naming [guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions) properties should use PascalCasing.

Reply:
ok!